tgs/src/test/cpp/tgs/FeatureSelection/CfsSubsetEvaluatorTest.cpp

```diff
@@ -88,7 +88,6 @@ namespace Tgs
   class CfsSubsetEvaluatorTest : public CppUnit::TestFixture
   {
     CPPUNIT_TEST_SUITE(CfsSubsetEvaluatorTest);
-    CPPUNIT_TEST(testGeneratedData20);
```

Comment:
Three lines below `testGeneratedData20` is added as a test, this is a duplicate.